js/apps/system/_admin/aardvark/APP/frontend/js/views/userPermissions.js

```diff
@@ -364,11 +364,11 @@
             if (databaseLevelDefaultPermission === 'undefined') {
               if (serverLevelDefaultPermission) {
                 if (serverLevelDefaultPermission === 'rw') {
-                  $('#' + database + '-db .mid > .readWrite').first().css('box-shadow', cssShadow);
+                  $('#' + window.CSS.escape(database) + '-db .mid > .readWrite').first().css('box-shadow', cssShadow);
                 } else if (serverLevelDefaultPermission === 'ro') {
-                  $('#' + database + '-db .mid > .readOnly').first().css('box-shadow', cssShadow);
+                  $('#' + window.CSS.escape(database) + '-db .mid > .readOnly').first().css('box-shadow', cssShadow);
                 } else if (serverLevelDefaultPermission === 'none') {
-                  $('#' + database + '-db .mid > .noAccess').first().css('box-shadow', cssShadow);
+                  $('#' + window.CSS.escape(database) + '-db .mid > .noAccess').first().css('box-shadow', cssShadow);
```

Comment:
> Do we also need to escape the db name in the template? MDN strongly advises to limit HTML IDs to `/^[a-z][-_0-9a-z]+$/` which probably doesn't need CSS escaping but is likely more strict than what we currently do in the template.

Yes, we need to escape the db name because it DB name allows UTF-8. That's why this was done.

> I'd normally also complain about the direct CSS manipulation instead of using class names, but as this is existing legacy code I'm ok with this for now as long as we don't add more styles.

if this was just styles, I wouldn't bother with the fix, but it denotes some information of significance. This is a bug fix.